lib/compiler/decorators.ts

```diff
@@ -1,26 +1,24 @@
+import { addTsdiMarker } from '../marker';
```

Comment:
Unrelated to this file, but I'd `console.warn` when an `@component` decorator is encountered in order to make users aware of it being legacy. `@managed` is a MUCH better name.